cruise.umple/src/UmpleInternalParser_CodeMixset.ump

```diff
@@ -0,0 +1,178 @@
+/*
+
+Copyright: All contributers to the Umple Project
+
+This file is made available subject to the open source license found at:
+http://umple.org/license
+
+This file analyze core umple meta model tokens such as which language to generate,
+the current namespace, etc and populates the umple meta model.
+
+Please refer to UmpleInternalParser.ump for more details.
+
+*/
+
+
+namespace cruise.umple.compiler;
+
+class UmpleModel{
+  depend java.util.stream.*;
+
+/*
+ * returns a mixset from UmpleModel based on its name.
+ * @Abdulaziz   
+ */
+ Mixset getMixset(String name) {
+	Mixset mixset = null;
+	for(MixsetOrFile mOrF : getMixsetsOrFiles(name))
+	{
+		if (mOrF instanceof Mixset)
```

Comment:
Best practice would be to have a Boolean something like 

```
Boolean isMixset = false;
```

is MixsetOrFile

and override the constructor in Mixset using after constructor to set the Boolean to true. Then you can simply say here mOrF.isMixset()

Reply:
I agree.